workshop/content/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md

```diff
@@ -52,6 +52,7 @@ The above code does several things:
 All that's left to get our pipeline up and running is to commit our changes and run one last cdk deploy. 
 
 ```
+git add .
 git commit -am "MESSAGE" && git push
```

Comment:
`git commit -a` already does `git add .` Why is the extra add required?

Reply:
I recall line 56, ```git commit -am``` combine the two flags was not working for on my container.  First tried to fix some file that was not added, then realized the``` -am``` flag not working. ``` -a -m``` worked for me. I don't know if that's just me or if anyone else has the similar problem.